package.json

```diff
@@ -13,13 +13,14 @@
     "Krishdevdb"
```

Comment:
hmm?

Reply:
Do you really wanna maintain this list?

Reply:
Yep

Reply:
Then you need to add a lot of other ppl as well 😅